requirements.txt

```diff
@@ -1,2 +1,9 @@
+cftime
```

Comment:
Please do not add all thee requirements. If we wanted to kerchunk FITS files, for example, we wouldn't need all these. You could add them to optional install selectors, to install like `pip install kerchunk[cdf,grib]`...